<launch>
	<node launch-prefix="xterm -e gdb --args" name="ipa_loc_feature_slam_development" type="ipa_loc_feature_slam_development" pkg="ipa_loc_feature_slam_development" />
</launch>
